internal/installcfg/installcfg_test.go

```diff
@@ -33,7 +33,7 @@ var _ = Describe("installcfg", func() {
 			BaseDNSDomain:          "redhat.com",
 			APIVip:                 "102.345.34.34",
 			IngressVip:             "376.5.56.6",
-			InstallConfigOverrides: `{"networking":{"networkType": "OVN-Kubernetes"},"fips":true}`,
+			InstallConfigOverrides: `{"networking":{"networkType": "OVNKubernetes"},"fips":true}`,
```

Comment:
Looks like a constant can be helpful here.

Reply:
OK